### PR TITLE
Allow more granular comments per mocks.

### PR DESCRIPTION
**Pull request by jmcarp**
Hoping to recruit @xtine for help with styling on this.

### CODE REVIEW

File: regulations/templates/regulations/node.html [hunk 1]
```diff
@@ -2,7 +2,12 @@
     Template for inner paragraphs of a reg section
 {% endcomment %}
 {% if node.header %}
-<h{{ node.list_level|add:"+3" }}>{{ node.header | safe }}</h{{ node.list_level|add:"+3" }}>
+<div>
+  <h{{ node.list_level|add:"+3" }} class="section-header">{{ node.header | safe }}</h{{ node.list_level|add:"+3" }}>
+  {% if root.node_type == "preamble" %}
+    <div class="activate-write" data-section="{{ node.markup_id }}">Comment on section {{ node.label_id }}</div>
+  {% endif %}
+</div>
```
Comment (cmc333333) [inline regulations/templates/regulations/node.html]: This is a bit messy. I wonder what a cleaner path would be?

File: regulations/templates/regulations/node.html [hunk 2]
```diff
@@ -13,6 +18,9 @@
     {{node.marked_up|safe}}
 {% endif %}
 </p>
+{% if root.node_type == "preamble" and node.node_type == "regtext" %}
+  <div class="activate-write" data-section="{{ node.markup_id }}">Comment on paragraph in section {{ node.label_id }}</div>
+{% endif %}
```
Comment (cmc333333) [inline regulations/templates/regulations/node.html]: Hrm. I don't think we should be having `regtext` nodes inside `preambles` -- I think this guard will need to change in the future. Maybe just `not node.title`

**Comment by cmc333333** [comment]
Didn't look super closely at the JS, but nothing stood out to me. @xtine, mind reviewing and merging?

File: regulations/templates/regulations/node.html (hunk 2 above)
Comment (jmcarp) [inline regulations/templates/regulations/node.html]: I may have misread, but there did seem to be `regtext` nodes nested within `preamble` nodes. Anyway, I'll verify that `not node.title` works and make the switch.

**Comment by jmcarp** [comment]
@cmc333333: does https://github.com/18F/regulations-site/pull/135/commits/6e3619f330722128b8d089dd07ad7ee2d56b1f9f handle your point about special-casing-related ugliness in `node.html`?

File: regulations/templates/regulations/node.html (hunk 1 above)
Comment (xtine) [inline regulations/templates/regulations/node.html]: Do we really need to explicitly set a header tag for this paragraph section?

**Comment by jmcarp** [comment]
@xtine: sorry, I think I clobbered your inline comment with the last commit. Anyway, this patch doesn't add the `<h{{ node.list_level|add:"+3" }}>` logic, if that's what you're getting at--we can refactor separately if you'd like. Or were you referring to a different change?

**Comment by xtine** [comment]
I think it's still in node.html but I'll merge in and we can refactor after.

**Pull request merged**